src/components/Tabs/Tabs.tsx

```diff
@@ -19,18 +20,31 @@ TabsList.displayName = TabsPrimitive.TabsList.displayName
 const TabsTrigger = forwardRef<
   ElementRef<typeof TabsPrimitive.Trigger>,
   ComponentPropsWithoutRef<typeof TabsPrimitive.Trigger>
->(({ className, ...props }, ref) => (
-  <TabsPrimitive.Trigger
-    ref={ref}
-    className={cn(
-      'data-[state=active]:rounded-md data-[state=active]:bg-primary',
-      'data-[state=inactive]:font-light data-[state=inactive]:text-disabled-primary',
-      'px-[16px] py-[8px]',
-      className,
-    )}
-    {...props}
-  />
-))
+>(({ className, onClick, value, ...props }, ref) => {
+  const router = useRouter()
+  const pathname = usePathname()
+  const searchParams = useSearchParams()
+
+  const updateSearchParam = (tab: string) => {
+    const currentParams = searchParams ? searchParams + '&' : ''
+    const updatedUrl = `${pathname}?${currentParams}tab=${tab}`
+    router.push(updatedUrl)
+  }
```

Comment:
Should we enclose this in a useEffect?

Reply:
why? it's not updating or depending on any local value.